core/arch/arm/plat-hikey/conf.mk

```diff
@@ -7,6 +7,7 @@ $(call force,CFG_PL011,y)
 $(call force,CFG_PM_STUBS,y)
 $(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
 $(call force,CFG_WITH_ARM_TRUSTED_FW,y)
+$(call force,CFG_CACHE_API,y)
```

Comment:
That should be `CFG_CACHE_API ?= y`, i.e., make it the default but still allow overriding with `make CFG_CACHE_API=n`.